pkg/scheduler/scheduler.go

```diff
@@ -125,7 +126,7 @@ func (s *Scheduler) onDelPod(obj interface{}) {
 }
 
 func (s *Scheduler) Start() {
-	kubeClient, err := k8sutil.NewClient()
+	kubeClient, err := client.NewClient()
```

Comment:
you can use `client.GetClient()` to get kubeClient, because this kubeClient having init function Initialization completed.

Reply:
Good idea. By doing so, some the unused code is also removed